deploy/host-operator/dev/toolchainconfig.yaml

```diff
@@ -6,6 +6,10 @@ spec:
   host:
     automaticApproval:
       enabled: true
+    deactivation:
+      deactivationDomainsExcluded: '@redhat.com'
+    registrationService:
+      environment: 'dev'
```

Comment:
I guess that you should specify also the `analytics` section:
```
      analytics:
        segmentWriteKey: 'test segment write key'
        woopraDomain: 'test woopra domain'
      environment: 'dev'
```
as well as the secret 
```
        secret:
          ref: 'host-operator-secret'
          twilioAccountSID: 'twilio.account.sid'
          twilioAuthToken: 'twilio.auth.token'
          twilioFromNumber: 'twilio.from_number'
```

Reply:
I believe the analytics config are only needed in the e2e environment? But I suppose the secret could be useful so I can add that.

Reply:
we use analytics in prod

Reply:
Added verification secret in bd095f6afb661d77d0eb790ae56cd9555fa3ff62

Reply:
@alexeykazakov yeah, I meant for the toolchainconfig.yaml in the toolchain-e2e repo which is only for dev and e2e environments. The ToolchainConfig for prod in sandbox-sre have the analytics set correctly so we should be good.